client_generic/Client/Voting.cpp

```diff
@@ -88,6 +88,13 @@ void	CVote::ThreadFunc()
 			//If we were interrupted before this point, we don't need to start the unnecessary transfer.
 			boost::this_thread::interruption_point();
 
+			// Convert "r3d0.sheepserver.net" to "v3d0.sheepserver.net" 
+			char *pChr = std::strchr(url, 'r');
```

Comment:
But the task is to change url directly for gold users. 
Scott said: "for those REST calls (voting and rendering), check the server URL and if it starts with "r" then replace it with "d""